Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Output.cpp

```diff
@@ -133,12 +133,14 @@ int CLuaFunctionDefs::OutputClientDebugString(lua_State* luaVM)
     unsigned char    ucRed = 255;
     unsigned char    ucGreen = 255;
     unsigned char    ucBlue = 255;
+    bool             bOmitDebugInfo = false;
```

Comment:
No more hungarian notation as per the new coding style.
(Should be confirmed first because [the wiki page](https://wiki.multitheftauto.com/wiki/Coding_guidelines#Style) still hasn't change since the decision was made)

Reply:
Why are we ditching Hungarian notation? Won't it make the codebase inconsistent?

Reply:
I really like hungarian notation to be honest. If it was only me, I would have keep it.
I can't remember if we decided to keep it depending of where the modifications are made to keep it consistent with the legacy code around.
@qaisjp and/or @botder ?

Reply:
I don't remember, the oldest thing I can find is https://discordapp.com/channels/278474088903606273/278474088903606273/531797552102768641

> I can't remember if we decided to keep it depending of where the modifications are made to keep it consistent with the legacy code around.

That sounds like it makes sense.

cc @multitheftauto/blue-contributors